tests/integration/labelbox_dev/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import uuid
+from labelbox_dev import Session, DataRow, Dataset
+
+
+@pytest.fixture(scope="session", autouse=True)
+def session(environ: str, testing_api_url, testing_api_key):
+    base_api_url = testing_api_url(environ)
+    api_key = testing_api_key(environ)
+    Session.initialize(base_api_url=base_api_url, api_key=api_key)
+
+
+@pytest.fixture(scope="session")
+def dataset():
+    dataset = Dataset.create({"name": "Test", "description": "Test dataset"})
+
+    yield dataset
+
+    dataset.delete()
+
+
+@pytest.fixture(scope="session")
+def datarows(dataset):
+    BASE_PATH = 'https://storage.googleapis.com/labelbox-datasets/image_sample_data/image-sample-'
+    data_rows = [{
+        "row_data": f"{BASE_PATH}{i}.jpg",
+        "global_key": f"{BASE_PATH}{i}.jpg" + str(uuid.uuid4()),
+    } for i in range(5)]
+    datarows = DataRow.create(dataset.id, data_rows)
```

Comment:
I've renamed to `data_rows` for consistency. This is a fixture so should it have assertions for data row creation? Wouldn't `DataRow.create` throw an exception if it was not able to create these data rows? I think we should add a separate test for data row creation.

Reply:
I think it's still good to verify that the test data is created successfully, so we can decouple test setup failures from actual test failures, but maybe it's not necessary in this case, since `DataRow.create` will throw an error anyway. As a general rule though, in cases where test data can be created incorrectly, I think it's good to have assertions.